content/docs/learning-guide/tutorials/_index.md

```diff
@@ -13,6 +13,15 @@ Learn the features, tools, and processes of **Open 3D Engine (O3DE)** with the h
 | - | - |
 | [2D Navigation with Kythera AI](ai/navigation-2d) | Create an artificial intelligence (AI) agent that performs ground-based navigation and follows a visually scripted behavior tree. This tutorial uses the broad toolset provided by the [Kythera AI](/docs/user-guide/gems/reference/kythera-ai) Gem. |
 
+## Custom tools
+
+Design and develop custom tools that extend the **O3DE Editor**. Tools are dockable widgets or dialog windows that add functionality to O3DE. You can create custom tools in C++ or Python.
+
+| Tutorial | Description |
+| - | - |
+| [Create a Custom Shape Tool in C++](custom-tools/shape-example-cpp.md) | Create a custom shape tool written in C++ that allows you to quickly create shape objects in your level. Learn about the **CppToolGem** template, the Tools UI API, and C++ development for O3DE. |
+| [Create a Custom Shape Tool in Python](custom-tools/shape-example-py.md) | Create a custom shape tool written in Python that allows you to quickly create shape objects in your level. Learn about the **PythonToolGem** template, the Tools UI API, and Python development for O3DE. |
```

Comment:
```suggestion
| [Create a Shape Tool Gem in Python](custom-tools/shape-example-py.md) | Create a custom gem written in Python that allows you to quickly create shape objects in your level. Learn about the **PythonToolGem** template, the Tools UI API, and Python development for O3DE. |
```

Reply:
I decided on the wording "Extend the O3DE Editor by creating a custom tool Gem" and "Create a custom tool Gem". See commit https://github.com/o3de/o3de.org/pull/1119/commits/d918c387c813f1ae29fc299a69f253b3c5a4c716.